code/modules/vending/wardrobes.dm

```diff
@@ -1,3 +1,6 @@
+#define MOTHROACH_START_CHANCE 5
```

Comment:
I mean, 5% isn't that low if it's on ran on every wardrobe vendor every round?
might make it so this becomes an every round thing

Reply:
Tramstation has 24 wardrobes, so perhaps a 1 or 2% chance would be more appropriate than 5%.

Reply:
To be fair, having the station feel like it is infested with mothroaches and they are a legit plague, but too cute for people to bring down the hammer, is a really funny world building.

Reply:
It's funny, but gimmicky downsides tend to become less funny over time when they're repeated almost every round for years on end.

Reply:
> Tramstation has 24 wardrobes, so perhaps a 1 or 2% chance would be more appropriate than 5%.

then that means that there would be a 24% chance for a wardrobe to be infested on a station. eeehhh... i'm wondering if we shouldn't either

A) lessen the probability
B) make it so that we have one warddrobe that's randomly set on initialize to be THE infested warddrobe (only one). can have one or zero wardrobes infested in a given shift. would preserve the gimmick a bit better imo

Reply:
> B) make it so that we have one warddrobe that's randomly set on initialize to be THE infested warddrobe (only one). can have one or zero wardrobes infested in a given shift. would preserve the gimmick a bit better imo

I think this is definitely the best solution to keep it consistent regardless of how big a map is and how many closets it contains (with optional station trait which increases the count)

Reply:
ill get back to this tomorrow with a solution that satisfies everyone hopefully